common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -512,5 +512,14 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = true,
   )
+  val PersonaliseSignInAfterCheckout = Switch(
```

Comment:
This is not a blocker, but wonder id we should add Gate to the switch name as it only applies to the Gate? ie. 'PersonaliseSignInGateAfterCheckout'

Reply:
Ah sorry I'd missed this comment. Yes that's a good idea - updated